packages/css/src/types.ts

```diff
@@ -1,15 +1,24 @@
 import type { MapLeafNodes, CSSVarFunction } from '@vanilla-extract/private';
-import type { PropertiesFallback, AtRule, Properties } from 'csstype';
+import type { Properties, AtRule } from 'csstype';
 
 import type { SimplePseudos } from './simplePseudos';
 
-type CSSTypeProperties = PropertiesFallback<number | (string & {})>;
+// csstype is yet to ship container property types as they are not in
+// the output MDN spec files yet. Remove this once that's done.
+interface ContainerProperties {
+  container?: string;
+  containerType?: 'size' | 'inline-size' | 'block-size' | (string & {});
```

Comment:
I can't see `block-size` in the spec (https://www.w3.org/TR/css-contain-3/#container-type) and if I try using it in Chrome, the dev tools marks it as an error.

Reply:
Yeah I got it from MDN. I might remove it if it isn't in the spec. You can still override with string values, this is just for autocomplete anyway.